Automattic/jetpack
# Boost: if a site is private treat it like it's offline

If a site is offline or private, then Boost can't receive data from WordPress.com, which includes pricing data for upgrades. Currently, this shows a blank screen on the Getting Started page.
This PR shows the upgrade table, but with missing prices and a warning message.


## Proposed changes:

* Add a notice to the pricing table.
* Remove `pricing` check before displaying the table.
* 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
* Make sure your site is private, or delete the cached JP option meta data with WP CLI: `wp user meta delete username my-jetpack-cache`
* Install this branch.
* Visit the plugin settings page.
* You should be shown the upgrade table. Prices will be missing.
